src/components/App/App.js

```diff
@@ -93,7 +98,7 @@ export default class App extends Component {
 				{!this.state.data ? <p>Loading...</p> : <p>SWAPI-box</p>}
 				{!this.state.data
 					? <p>No Cards yet</p>
-					: <CardContainer data={this.state.data} />}
+					: <CardContainer peopleArray={this.state.data[0]} />}
```

Comment:
Here I just passed the peopleArray into the card container because I'm just rendering the people so far